Assets/Scripts/MVC/Tank/TankView.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TankView : MonoBehaviour
```

Comment:
Use this class to take the input and physics as per the need and pass to the controller for doing its functionality.